glossary.yml

```diff
@@ -291,6 +291,11 @@
     def: >
       A [join](#join) that keeps rows from table A whose keys do *not* match keys in
       table B.
+  af:
+    term: "anti-koppeling"
+    def: >
+      'n [Koppeling](#join) wat rye in tabel A bevat wie se sleutels *nie* die
```

Comment:
@jsteyn what about "n koppeling wat rye in table A hou wie se sleutels *nie* gevind word in table B nie"

Reply:
Ek stem saam